packages/backend/src/helpers/flow-templates.ts

```diff
@@ -18,6 +21,19 @@ import logger from './logger'
 
 // e.g. {{user_email}}, {{tile_col_data.Email}}
 const PLACEHOLDER_REGEX = /\{\{[a-zA-Z0-9_.? ]+\}\}/
+const FORM_ID_LENGTH = 24
+const CREATE_APP_EVENT_KEY = (appKey?: string, eventKey?: string) =>
+  `${appKey}_${eventKey}`
+// chose app_event key since it is unique among all triggers and actions
+const HELP_MESSAGE_APP_EVENT_KEY_SET = new Set([
+  'formsg_newSubmission',
+  'tiles_createTileRow',
+  'tiles_findSingleRow',
+  'tiles_updateSingleRow',
+  'postman_sendTransactionalEmail',
+  'slack_sendMessageToChannel',
+  'telegram-bot_sendMessage',
+])
```

Comment:
dont do this - instead minimise the need to synchronise your frontend constants and your backend. you could already check the existence of these keys on the frontend and display/hide accordingly

Reply:
ok changed thks